### PR TITLE
Tasks css

### PR DESCRIPTION
**Pull request by psykohack**

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hi
```
Comment (dmorina) [inline test.txt]: remove this file please

File: static/templates/base/top_navigation.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<div class="md-primary-custom" style="width:100%;height:48px;">
+    <div style="position:absolute;left:32px;top:4px;">
+      <img ng-src="{{user.picture || 'static/images/logo_top_bar.png' }}" alt="Daemo">
+    </div>
+    <div style="position:absolute;right:32px;width:650px;">
+      <md-tabs md-dynamic-height="true" md-selected="data.selectedIndex" md-align-tabs="top">
+        <md-tab id="tab1" ng-click="ctrl.navigateTo('profile')">
+          <md-tab-label>My Account</md-tab-label>
+          <md-tab-body></md-tab-body>
```
Comment (dmorina) [inline static/templates/base/top_navigation.html]: you could remove body tags

File: static/templates/base/top_navigation.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<div class="md-primary-custom" style="width:100%;height:48px;">
+    <div style="position:absolute;left:32px;top:4px;">
```
Comment (dmorina) [inline static/templates/base/top_navigation.html]: use toolbars here, it will give you the proper height too

File: static/css/top-navigator.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    font-weight: bold;
+}
+
+.md-primary-custom {
+  background-color: #FBF5AB;
+}
+
+.right-tabs {
+	float: right;
+}
```
Comment (dmorina) [inline static/css/top-navigator.css]: use scss please

**Comment by psykohack** [comment]
New changes in the same branch (psykohack/crowd-research/tasks-css).
test.txt file was deleted.
Now I'm using SCSS instead CSS
md-toolbar component added.
Unused tags (tab-body) were deleted from the top_menu HTML file
Menu item colors now are the same to the master design template.

I'm Carlos Toxtli (ctoxtli@gmail.com in Slack) and I'm collaborating with Saiph and Claudia.

**Comment by dmorina** [comment]
remove  static/css/.sass-cache/d2e792c91a618ca7241a1e62b41213f9cc2e584d/top-navigator.scssc
and the text file please

**Comment by psykohack** [comment]
.sass-cache directory deleted

**Comment by dmorina** [comment]
test.txt too, please

**Comment by psykohack** [comment]
test.txt file deleted

File: test.txt (hunk 1 above)
Comment (neilthemathguy) [inline test.txt]: This is still pending, could you delete this and re-raise the PR.

**Comment by psykohack** [comment]
test.txt file deleted (again) but this time it looks that worked.

**Comment by dmorina** [comment]
closing until we get the go from Karolina

**Pull request closed without merging**